src/tests/MedStickerEncryptionCharlie.test.js

```diff
@@ -6,11 +6,8 @@ import {
   fingerprint
```

Comment:
this whole file needs to be changed as part of the https://github.com/VivyTeam/krypt-web/pull/75/files#r299435041